app/forms/nsm/send_back_form.rb

```diff
@@ -23,7 +23,7 @@ def save
 
       claim.with_lock do
         update_local_data
-
+        AppStoreClient.new.unassign(claim)
```

Comment:
So what should happen is if the API call fails this will raise an error, and the `with_lock` block will pick up on the error and roll back the transaction and release the lock

Reply:
Unwarranted concern, still thinking in terms of "what if we update the local data and the app store but an event wipes it"